Modules/+Experiments/+AutoExperiment/@AutoExperiment_invisible/run.m

```diff
@@ -87,8 +87,12 @@ function run(obj,status,managers,ax)
                 prev_mask = and(mask,[obj.data.sites(site_index).experiments.continued]==last_attempt); % Previous run
                 % Check redo flag for this site and this experiment
                 redo = false;
-                if ~isempty(obj.analysis)
-                    redo = obj.analysis.sites(site_index,exp_index).redo;
+                if ~isempty(obj.analysis) && ~isempty(obj.analysis.sites(site_index,exp_index).redo)
+                    try
+                        redo = logical(obj.analysis.sites(site_index,exp_index).redo);
```

Comment:
It sounds like there's an expected potential case in which this errors - what happens?

Reply:
I added that when adding the logical conversion. Not sure if that errors on anything - but you're right it would be better to have asserts there instead.